src/client/client.ts

```diff
@@ -4,6 +4,7 @@
  */
 
 import { SqlQuerySpec } from '@azure/cosmos';
+import { TokenCredential } from '@azure/identity';
```

Comment:
nit: use type import where you pull in `@azure/identity` to ensure you're not depending on the code
```suggestion
import type { TokenCredential } from '@azure/identity';
```